auction/src/components/PageLayout/NavBar/NavBar.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import { NavLink, Link, useHistory } from 'react-router-dom';
+
+import Wrapper from '../../Common/Wrapper/Wrapper';
+import SearchBar from './SearchBar';
+
+import {
+    navBar,
+    navLinks,
+    logo
+} from './NavBar.module.scss';
+
+const NavBar = () => {
+    const history = useHistory();
+
+    const handleSearch = search => {
+        search = search.trim().split(' ').map(word => word.trim()).filter(word => word !== '').join(',');
```

Comment:
New line for each function call. Also dont change argument you are passing here, create new variable and use that.